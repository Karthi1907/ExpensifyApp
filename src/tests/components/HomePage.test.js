import React from 'react';
import ExpenseDashboardPage from '../../components/HomePage';
import { shallow } from 'enzyme';

test('HomePage rendering test case', () => {

    const wrapper = shallow( <ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();

})
