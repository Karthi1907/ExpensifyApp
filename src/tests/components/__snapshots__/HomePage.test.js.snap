// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage rendering test case 1`] = `
<div>
  This is from my dashboard component
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
